src/main/java/MongoDBDAO.java

```diff
@@ -1,139 +1,140 @@
-import com.mongodb.ConnectionString;
```

Comment:
Have a look at https://github.com/diplomit-dtu/diplomBackend/blob/master/src/main/java/data/mongoImpl/MongoCourseDAO.java for a simpler approach. Some things are overcomplicated